frontend/src/views/reports/ReportingView.vue

```diff
@@ -167,196 +46,155 @@
+          const monthName = moment(month).format('MMMM')
+          const monthId = this.getMonthIdByName(monthName)
+          const fiscalYearId = this.getFiscalYearIdByDate(month)
+          const fiscalYearName = this.getFiscalYearNameById(fiscalYearId)
+          const surveyResponse = this.surveyResponses?.find((item) => item.facilityId === facility.facilityId && item.fiscalYearId === fiscalYearId && item.reportingMonthId === monthId)
+          const isPreviousMonth = moment().startOf('month').isAfter(moment(month))
+          const hasNotSubmittedResponse = isEmpty(surveyResponse) || surveyResponse?.stateCode === CRM_STATE_CODES.ACTIVE
+          if (isPreviousMonth && hasNotSubmittedResponse) {
+            pendingReports.push({
+              contactId: this.userInfo?.contactId,
+              surveyResponseId: surveyResponse?.surveyResponseId,
+              surveyResponseReferenceNumber: surveyResponse?.surveyResponseReferenceNumber ?? BLANK_FIELD,
+              alertType: this.getSurveyResponseAlertType(month),
+              title: `Monthly Report - ${monthName} ${fiscalYearName}`,
+              facilityId: facility.facilityId,
+              facilityName: facility.facilityName,
+              reportingMonthId: monthId,
+              reportingMonthName: monthName,
+              fiscalYearId: fiscalYearId,
+              status: !isEmpty(surveyResponse) ? 'Draft' : BLANK_FIELD,
+              latestActivity: surveyResponse?.latestActivity ? format.formatDate(surveyResponse?.latestActivity) : BLANK_FIELD,
+            })
+          }
+        })
+      })
+      return pendingReports
     },
 
-    formattedEndDate: {
-      get() {
-        return this.formatDate(this.endDate)
-      },
-      set(val) {
-        this.endDate = this.parseDateString(val)
-      },
-    },
-
-    defaultFiscalYear() {
-      return this.fiscalYears?.find((fiscalYear) => fiscalYear.statusCode === 1)
-    },
-
-    defaultReportingMonth() {
-      const currentMonthName = new Date().toLocaleString('en-ca', { month: 'long' })
-      return this.months?.find((month) => month.name === currentMonthName)
-    },
-
-    surveyResponseType() {
-      let responseType = SURVEY_RESPONSE_TYPES.MONTHLY
-      if (['November'].includes(this.selectedReportingMonth?.name)) {
-        responseType = SURVEY_RESPONSE_TYPES.QUARTERLY
-      } else if (['December'].includes(this.selectedReportingMonth?.name)) {
-        responseType = SURVEY_RESPONSE_TYPES.BI_ANNUAL
-      }
-      // TODO (vietle-cgi) Add ANNUAL type when we know which month to use
-      return responseType
+    submittedReports() {
+      const submittedReports = this.surveyResponses?.filter((response) => response.stateCode === CRM_STATE_CODES.INACTIVE)
+      submittedReports?.forEach((response) => {
+        response.submittedDate = format.formatDate(response?.endDate)
+        response.status = response?.isSubmittedLate ? 'Completed - Late' : 'Completed'
```

Comment:
Maybe add these values if we're creating a constant for statuses.

Reply:
I've added constant for survey_response_statuses